packages/hotspot/configure/src/__tests__/index.test.js

```diff
@@ -39,161 +38,131 @@ const model = () => ({
   partialScoring: false
 });
 
-describe('Root', () => {
-  let w;
+jest.mock('react-dom', () => ({
+  render: jest.fn()
+}));
+
+describe('index', () => {
+  let Def;
+  let el;
   let onModelChanged = jest.fn();
   let initialModel = model();
-  let modelCopy;
 
-  const wrapper = extras => {
-    const defaults = {
-      onModelChanged,
-      model: model()
-    };
-    const props = { ...defaults, ...extras };
-
-    return shallow(<Root {...props} />);
-  };
+  beforeAll(() => {
+    Def = require('../index').default;
+  });
 
   beforeEach(() => {
-    modelCopy = model();
+    el = new Def();
+    el.model = initialModel;
+    el.onModelChanged = onModelChanged;
   });
 
-  describe('snapshot', () => {
-    it('renders', () => {
-      w = wrapper();
-      expect(w).toMatchSnapshot();
+  describe('set model', () => {
+    it('calls ReactDOM.render', () => {
+      expect(ReactDOM.render).toHaveBeenCalled();
```

Comment:
@edeustace here I finally went with this solution, before I've managed to implement something like

```
expect(ReactDOM.render).toHaveBeenCalledWith(React.createElement('Root', { 
  configure: el._configure,
  disabled: el._disabled,
  model: el._model,
  onColorChanged: el.onColorChanged,
  onImageUpload: el.onImageUpload,
  onMultipleCorrectChanged: el.onMultipleCorrectChanged,
  onPartialScoringChanged: el.onPartialScoringChanged,
  onPromptChanged: el.onPromptChanged,
  onRemoveShape: el.onRemoveShape,
  onUpdateImageDimension: el.onUpdateImageDimension,
  onUpdateShapes: el.onUpdateShapes,
}), el)
```

tho 

> Compared values have no visual difference

was appearing, this test was failing so I switched to `toHaveBeenCalled()` method. 

Let me know if looks OK to you.